docs/current_docs/quickstart/ci/index.mdx

```diff
@@ -0,0 +1,253 @@
+---
+slug: /ci/quickstart
+hide_table_of_contents: false
+title: "Build a CI Pipeline"
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Build a CI Pipeline
+
+This quickstart will guide you through building a CI pipeline for an application using Dagger.
+
+## Requirements
+
+This quickstart will take you approximately 25 minutes to complete. You should be familiar with programming in Go, Python,  TypeScript, PHP, or Java.
+
+Before beginning, ensure that:
+- you have [installed the Dagger CLI](../../install.mdx).
+- you know [the basics of Dagger](../basics/index.mdx).
+- you have [Git](https://git-scm.com/downloads) and a container runtime installed on your system and running. This can be [Docker](https://docs.docker.com/engine/install/), but you can also use Podman, nerdctl, or other Docker-like systems.
+- you have a GitHub account (optional, only if configuring Dagger Cloud)
+
+## Get the example application
+
+The example application is a skeleton Vue framework application that returns a "Hello from Dagger!" welcome page. Clone its repository and set it as the current working directory:
+
+```shell
+git clone https://github.com/dagger/hello-dagger
+cd hello-dagger
+```
+
+## Visualize in Dagger Cloud (optional)
+
+:::important
+This step is optional and will create a Dagger Cloud individual plan account. The individual plan is free of charge for a single user. You will require a GitHub account for account setup and identity verification. If you prefer not to sign up for Dagger Cloud, you can skip this section.
```

Comment:
Maybe we should shorten this (e.g. remove the pricing and requirement for github) and link to the dagger.io/cloud page.

Reply:
done